src/renderer/src/helpers/StringHelper.ts

```diff
@@ -25,4 +25,17 @@ export class StringHelper {
   static normalizeText(text: string) {
     return text.trim().toLowerCase()
   }
+
+  static validateName(name: string, maxLength: number = 30) {
+    const trimmedName = name.trim()
+    const isNameEmpty = trimmedName.length === 0
+    const isNameTooLong = trimmedName.length > maxLength
```

Comment:
1. Use `trimmedValue` instead of `trimmedName`.
2. Use `isEmpty` instead of `isNameEmpty`.
2. Use `isTooLong` instead of `isNameTooLong`.